docs/book/src/developer/core/tilt.md

```diff
@@ -20,14 +20,70 @@ workflow that offers easy deployments and rapid iterative builds.
 ## Getting started
 
 ### Create a kind cluster
-A script to create a KIND cluster along with a local Docker registry and the correct mounts to run CAPD is included in the hack/ folder.
 
-To create a pre-configured cluster run:
+The following CAPI infrastructure providers are suitable for local development:
```

Comment:
Any provider can be used with tilt. This doc is not only about local development.

I e.g. use the capd kind script + a different tilt-settings.yaml to develop CAPV, but also for CAPA or CAPZ in the past.